src/redux/api/ordersApi.tsx

```diff
@@ -0,0 +1,17 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { environment } from "../../constants/environment";
+
+export const ordersApi = createApi({
+  reducerPath: "ordersApi",
+  baseQuery: fetchBaseQuery({ baseUrl: environment.apiBaseUrl }),
+  tagTypes: ["orders"],
+  endpoints: (builder) => ({
+    getAllOrders: builder.query({
+      query: ({ limit = 10, offset = 0, optionalQuery = "", searchQuery = "" }) =>
```

Comment:
why need to set default value?

Reply:
after tested, seems don't need them, will delete them